max_range: 0.65
